client/my-sites/media-library/header.jsx

```diff
@@ -119,6 +120,20 @@ export default React.createClass( {
 		} );
 	},
 
+	renderUploadAndEditButton() {
+		if ( ! config.isEnabled( 'image-editor' ) ) {
+			return;
+		}
+
+		return (
+			<button className="button is-desktop"
```

Comment:
@aduth No plans for mobile devices right now. I think it should be responsive for desktop, but on mobile web I don’t know if it makes sense. People have direct access to the mobile native photo editing tools.